tests/test_api.py

```diff
@@ -144,6 +146,33 @@ def test_webhook_post(app, tester_id, access_token, receiver):
             )
 
 
+def test_webhook_post_no_token(app, tester_id, receiver):
+    ds = app.extensions['security'].datastore
+
+    with app.test_request_context():
+        user = ds.get_user(tester_id)
+        with app.test_client() as client:
+            # Manual login via view
+            response = client.post(
+                url_for_security('login'),
+                data={'email': user.email, 'password': user.password},
+                environ_base={'REMOTE_ADDR': '127.0.0.1'}
+            )
+
+            assert response.status_code == 302
+            assert user.get_id() == current_user.get_id()
+
+            payload = dict(somekey='somevalue')
+            make_request(
+                None,
+                client.post,
```

Comment:
can you also make a `client.get` to check https://github.com/inveniosoftware/invenio-webhooks/pull/48#issuecomment-262193592 ?